browser/ui/webui/speedreader/speedreader_toolbar_data_handler_impl.cc

```diff
@@ -334,3 +325,7 @@ void SpeedreaderToolbarDataHandlerImpl::OnNativeThemeUpdated(
 void SpeedreaderToolbarDataHandlerImpl::OnTuneBubbleClosed() {
   events_->OnTuneBubbleClosed();
 }
+
+void SpeedreaderToolbarDataHandlerImpl::OnContentsReady() {
+  active_tab_helper_->OnToolbarStateChanged(current_button_);
```

Comment:
i'm a bit lost in understanding when `active_tab_helper_` can be nullptr and we it cannot. Can we CHECK() here ?
You could mention it's null/non-null state in a comment